src/tiling/tiler.rs

```diff
@@ -460,9 +460,9 @@ pub mod test {
     assert_eq!((32, 0, 32, 32), rect(&tile.planes[2]));
 
     let tile = &tile_states[2].rec; // the top-right tile
```

Comment:
Change comments above.

Reply:
Sure, which format of doc would you think best?  New *.md file under doc directory?

Reply:
These are the comments I meant. https://github.com/xiph/rav1e/pull/2445/files#diff-61a9dba6df92b3be4f74e67feaa6cd10R448

I think a new md file would be great.

Reply:
I've pushed document for this PR, under name "doc/TILE_ON_FRAME_BOUNDARY.md"
Please let me know if you'd like more revisions or it looks fine for landing.

Reply:
@KyleSiefring is that enough for you?